src/components/app-img/app-img.tsx

```diff
@@ -57,7 +60,7 @@ export class Img {
   }
 
   private changeImageFormat() {
-    if (this.loadSrc && localStorage && localStorage.getItem('allowWebp') === 'true') {
+    if (this.loadSrc && 1 > 2 && (this.isServer || localStorage.getItem('allowWebp') === 'true')) {
```

Comment:
why is this 1 > 2 thing? To ensure some conditions?

Reply:
Ah, this was leftover from when I was testing something. Good catch, I will fix.

Reply:
Fixed